server/src/main/java/com/objectcomputing/checkins/services/onboard/signrequest/SignRequestController.java

```diff
@@ -0,0 +1,55 @@
+package com.objectcomputing.checkins.services.onboard.signrequest;
+
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.http.HttpRequest;
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.*;
+import io.micronaut.http.client.HttpClient;
+import io.micronaut.http.client.annotation.Client;
+import jakarta.inject.Inject;
+
+import io.micronaut.security.annotation.Secured;
+import io.micronaut.security.rules.SecurityRule;
+import io.micronaut.security.authentication.Authentication;
+
+import javax.print.attribute.standard.Media;
+import java.util.Arrays;
+import java.util.StringTokenizer;
+import java.util.*;
+
+@Secured(SecurityRule.IS_ANONYMOUS)
+@Controller
+public class SignRequestController {
+    @Inject
+    @Client("https://ocitest.signrequest.com/api/v1/")
+    private HttpClient httpClient;
+    private Map<String,String> signerInfo = new HashMap<>();
+
+    @Get("/signer")
+    public String getData(){
+
+        try{
+            String retrieve = httpClient.toBlocking()
+                    .retrieve(HttpRequest.GET("/documents/152408ef-4b71-40a4-9416-92011a443450/")
+                            .header("Authorization","Token 8e3b636de707468e40abd0d8e212c2507e5d2197"));
```

Comment:
We do not want to store auth tokens directly ever if possible. Can this be used as a .gitignore'd local variable set during run/build, and referenced here instead by variable?